backend/coins/btc/transaction_test.go

```diff
@@ -44,6 +46,65 @@ func mustKeypath(t *testing.T, keypath string) signing.AbsoluteKeypath {
 	return kp
 }
 
+type MockTransaction struct {
+	address *addresses.AccountAddress
+}
+
+func (m MockTransaction) Balance() (*accounts.Balance, error) {
+	return nil, nil
+}
+
+func (m MockTransaction) Transactions(isChange func(blockchain.ScriptHashHex) bool) (accounts.OrderedTransactions, error) {
+	return nil, nil
+}
+
+func (m MockTransaction) UpdateAddressHistory(scriptHashHex blockchain.ScriptHashHex, txs []*blockchain.TxInfo) {
+}
+
+func (m MockTransaction) SpendableOutputs() (map[wire.OutPoint]*transactions.SpendableOutput, error) {
+	return map[wire.OutPoint]*transactions.SpendableOutput{
+		*wire.NewOutPoint(&chainhash.Hash{}, 0): {TxOut: wire.NewTxOut(1000000000, m.address.PubkeyScript())},
+	}, nil
+}
+
+func (m MockTransaction) Close() {}
+
+func testAccount(t *testing.T, config *config.Account) *Account {
+	t.Helper()
+	account := mockAccount(t, config)
+	account.coin.TstSetMakeBlockchain(func() blockchain.Interface {
+		return &blockchainMocks.BlockchainMock{
+			MockRelayFee: func() (btcutil.Amount, error) {
+				return btcutil.Amount(1001), nil
+			},
+			MockEstimateFee: func(number int) (btcutil.Amount, error) {
+				switch number {
+				case 2:
+					return btcutil.Amount(10e7), nil
+				case 6:
+					return btcutil.Amount(10e6), nil
+				case 12:
+					return btcutil.Amount(10e5), nil
+				case 24:
+					return btcutil.Amount(10e4), nil
+				default:
+					return btcutil.Amount(10e6), nil
+				}
+			},
+		}
+	})
+	account.coin.blockchain = account.coin.makeBlockchain()
+	require.NoError(t, account.Initialize())
+	account.ensureAddresses()
```

Comment:
this is called by Initialize() already, so could be skipped? Seems too low level to have to deal with in a unit test.

Reply:
Good catch, done